core/tests/protractor/accessibility.js

```diff
@@ -19,11 +19,37 @@
 
 var general = require('../protractor_utils/general.js');
 var waitFor = require('../protractor_utils/waitFor.js');
+var users = require('../protractor_utils/users.js');
 
 var LibraryPage = require('../protractor_utils/LibraryPage.js');
 
 describe('screenreader and keyboard user accessibility features', function() {
   var libraryPage = null;
+  var timeout = 15000;
+  var getStartedUrl = 'http://localhost:9001/get-started';
+  var communityLibraryUrl = 'http://localhost:9001/community-library';
+  var learnerDashboardUrl = 'http://localhost:9001/learner-dashboard';
+  var creatorDashboardUrl = 'http://localhost:9001/creator-dashboard';
+  var aboutUrl = 'http://localhost:9001/about';
+  var notificationsUrl = 'http://localhost:9001/notifications';
+  var preferencesUrl = 'http://localhost:9001/preferences';
+
+  var triggerKeys = async function(key) {
```

Comment:
This name doesn't suggest that we're holding down CTRL along with the key

Reply:
Done